packages/vscode-extension/src/commandHandlers/connection.ts

```diff
@@ -76,6 +80,74 @@ export function createConnectionPanel(): void {
   ConnectionPanel.createOrShow(context.extensionPath, undefined, '');
 }
 
+//The npm data object contains more fields from the npm registrys package.json, but we only need dist-tags here
+type NpmData = {
+  'dist-tags'?: Record<string, string>;
+};
+
+export type NpmRelease = {
+  tag: string;
+  version: string;
+};
+
+export async function getTaggedRegistryVersions(): Promise<NpmRelease[]> {
+  const registryUrl = 'https://registry.npmjs.org/@neo4j-cypher/lint-worker';
+  const tagToFilter = 'neo4j-5.20.0';
+
+  try {
+    const response = await axios.get<NpmData>(registryUrl);
+    const data: NpmData = response.data;
+    const taggedVersions: { tag: string; version: string }[] = [];
+    if (data !== null && data['dist-tags'] !== null) {
+      for (const [tag, version] of Object.entries(data['dist-tags'])) {
+        if (
+          typeof tag === 'string' &&
+          typeof version === 'string' &&
+          tag === tagToFilter
+        ) {
+          taggedVersions.push({ tag, version });
+        }
+      }
+    }
+
+    return taggedVersions;
+  } catch (error) {
+    return [];
+  }
+}
```

Comment:
This method should return all versions as well, not just 5.20.0, for when we have more linters